src/runtime_src/hip/api/hip_memory.cpp

```diff
@@ -229,6 +229,32 @@ namespace xrt::core::hip
     s_hdl->enqueue(command_cache.get(cmd_hdl));
   }
 
+  // fill data to dst async.
+  template<typename element_type> static void
+  hip_memset_async(void* dst, int value, size_t size, hipStream_t stream)
```

Comment:
Second argument should be `element_type value`

Reply:
fixed.